src/atoms/Icon/Readme.md

```diff
@@ -18,7 +18,7 @@ _xIcon example with full SVG DOM and css hover:_
 // You can't edit this. Sorry.
```

Comment:
??

Reply:
I think this comment was to distinguish from the editable code samples throughout the RCL documentation